deployment/gke-single-cluster/flyte_generated.yaml

```diff
@@ -0,0 +1,1550 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: sparkoperator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: sparkoperator
+subjects:
+- kind: ServiceAccount
+  name: sparkoperator
+  namespace: sparkoperator
+---
+apiVersion: v1
+data:
+  aa_namespace.yaml: |
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: {{ namespace }}
+    spec:
+      finalizers:
+      - kubernetes
+kind: ConfigMap
+metadata:
+  name: clusterresource-template-dgc9fcm2kh
+  namespace: flyte
+---
+apiVersion: v1
+data:
+  datacatalog_config.yaml: |
+    logger:
+      show-source: true
+      level: 5
+    datacatalog:
+      storage-prefix: metadata/datacatalog
+      metrics-scope: "datacatalog"
+      profiler-port: 10254
+    application:
+      grpcPort: 8089
+    storage:
+      connection:
+        access-key: minio
+        auth-type: accesskey
+        disable-ssl: true
+        endpoint: http://minio.flyte.svc.cluster.local:9000
+        region: us-east-1
+        secret-key: miniostorage
+      cache:
+        max_size_mbs: 10
+        target_gc_percent: 100
+      container: my-container
+      type: minio
+    database:
+      port: 5432
+      username: postgres
+      host: postgres
+      dbname: datacatalog
+      options: sslmode=disable
+kind: ConfigMap
+metadata:
+  name: datacatalog-config-5g5bbg2k49
+  namespace: flyte
+---
+apiVersion: v1
+data:
+  flyteadmin_config.yaml: |
+    logger:
+      show-source: true
+      level: 5
+    application:
+      httpPort: 8088
+      grpcPort: 8089
+    flyteadmin:
+      roleNameKey: "iam.amazonaws.com/role"
+      profilerPort: 10254
+      metricsScope: "flyte:"
+      metadataStoragePrefix:
+        - "metadata"
+        - "admin"
+      testing:
+        host: http://flyteadmin
+    database:
+      # Create a database like postgres and override these values
+      port: 5432
+      username: postgres
+      password: awesomesauce
+      # Recommended to use passwordPath and mount it using kubescrets or the like
+      # passwordPath: "/var/run/CREDENTIALS_DB_PASSWORD"
+      # host here is the private ip address of the CloudSQL Db in private mode
+      host: 10.23.0.3
+      dbname: flyte
+      options: sslmode=disable
+    storage:
+      type: stow
+      stow:
+        kind: google
+        config:
+          scopes: ""
+          project_id: flyte-sandbox
+          json: ""
+      container: "flyte-sandbox"
+    task_resources:
+      defaults:
+        cpu: 200m
+        gpu: 0
+        memory: 500Mi
+        storage: 100Mi
+      limits:
+        cpu: 62
+        gpu: 8
+        memory: 256Gi
+        storage: 5Gi
+    domains:
+      - id: development
+        name: development
+      - id: staging
+        name: staging
+      - id: production
+        name: production
+      - id: domain
+        name: domain
+    registration:
+      maxWorkflowNodes: 100
+    scheduler:
+      eventScheduler:
```

Comment:
should we remove this section completely or is there a gcp equivalent we can move to?